infrastructure/modules/aws/eks/outputs.tf

```diff
@@ -1,26 +1,31 @@
 output "arn" {
   description = "ARN of EKS cluster"
-  value       = data.aws_eks_cluster.cluster.arn
+  value       = module.eks.cluster_arn
 }
+
 output "cluster_endpoint" {
   description = "Endpoint for EKS control plane"
   value       = module.eks.cluster_endpoint
 }
 
-output "certificate_authority" {
-  description = "Endpoint for EKS control plane"
-  value       = data.aws_eks_cluster.cluster.certificate_authority
+output "aws_eks_module" {
+  description = "aws eks module"
+  value       = module.eks
+}
+
+output "cluster_certificate_authority_data" {
+  description = "cluster_certificate_authority_data"
+  value       = module.eks.cluster_certificate_authority_data
 }
 
-output "token" {
-  description = "Authentication token for EKS"
-  value       = data.aws_eks_cluster_auth.cluster.token
-  sensitive   = true
+output "cluster_name" {
+  description = "cluster_name"
+  value       = module.eks.cluster_name
 }
 
 output "name" {
```

Comment:
Delete this output (cf. cluster_name)

Reply:
ok

Reply:
hum not sure do we need both cluster_name and cluster_id ? or only one of them ?